packages/multimedia/libdvdnav/package.mk

```diff
@@ -0,0 +1,44 @@
+################################################################################
+#      This file is part of OpenELEC - http://www.openelec.tv
+#      Copyright (C) 2009-2016 Stephan Raue (stephan@openelec.tv)
+#
+#  OpenELEC is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  OpenELEC is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with OpenELEC.  If not, see <http://www.gnu.org/licenses/>.
+################################################################################
+
+PKG_NAME="libdvdnav"
+PKG_VERSION="43b5f81"
+PKG_REV="1"
+PKG_ARCH="any"
+PKG_LICENSE="GPL"
+PKG_SITE="https://github.com/xbmc/libdvdnav"
+PKG_URL="https://github.com/xbmc/libdvdnav/archive/$PKG_VERSION.tar.gz"
+PKG_DEPENDS_TARGET="toolchain libdvdread"
+PKG_PRIORITY="optional"
+PKG_SECTION="multimedia"
+PKG_SHORTDESC="libdvdnav: a library that allows easy use of sophisticated DVD navigation features such as DVD menus, multiangle playback and even interactive DVD games."
+PKG_LONGDESC="libdvdnav is a library that allows easy use of sophisticated DVD navigation features such as DVD menus, multiangle playback and even interactive DVD games."
+
+PKG_IS_ADDON="no"
+PKG_AUTORECONF="yes"
+
+PKG_CONFIGURE_OPTS_TARGET="--disable-static"
+
+pre_configure_target() {
+  export CFLAGS="-D_XBMC -DHAVE_DVDCSS_DVDCSS_H $CFLAGS"
+}
+
+post_makeinstall_target() {
+  mkdir -p $INSTALL/usr/lib/kodi/system/players/VideoPlayer/
+    ln -s ../../../../libdvdnav.so $INSTALL/usr/lib/kodi/system/players/VideoPlayer/libdvdnav-$TARGET_ARCH-linux.so
```

Comment:
You can replace `../../../../libdvdnav.so` with `/usr/lib/libdvdnav.so` which is less ambiguous (but won't be required at all if building static).